ci/provisioning/post_provision_config_common_functions.sh

```diff
@@ -191,13 +191,14 @@ set_local_repo() {
     local version
     version="$(lsb_release -sr)"
     version=${version%%.*}
-    if [ "$repo_server" = "artifactory" ] &&
-       [ -z "$(rpm_test_version)" ] &&
-       [[ ${CHANGE_TARGET:-$BRANCH_NAME} != weekly-testing* ]]; then
-        # Disable the daos repo so that the Jenkins job repo or a PR-repos*: repo is
-        # used for daos packages
-        dnf -y config-manager \
-            --disable daos-stack-daos-"${DISTRO_GENERIC}"-"$version"-x86_64-stable-local-artifactory
+    if [ "$repo_server" = "artifactory" ] && [ -z "$(rpm_test_version)" ]; then
+        if [[ ${CHANGE_TARGET:-$BRANCH_NAME} != weekly-testing* ]] ||
+           [[ ${CHANGE_TARGET:-$BRANCH_NAME} != provider-testing* ]]; then
```

Comment:
FWIW. rather than adding another level of nesting, I think this could have been:

```suggestion
    if [ "$repo_server" = "artifactory" ] && [ -z "$(rpm_test_version)" ] &&
         { [[ ${CHANGE_TARGET:-$BRANCH_NAME} != weekly-testing* ]] ||
           [[ ${CHANGE_TARGET:-$BRANCH_NAME} != provider-testing* ]]; }; then
```